packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState, createContext } from 'react';
 import styled from 'styled-components';
 import { connect } from 'react-redux';
 import { colors, variables } from '@trezor/components';
-import SuiteNotifications from '@suite-components/Notifications';
+import SuiteBanners from '@suite/components/suite/Banners';
```

Comment:
nitpick, from `@suite-components/Banners`

Reply:
one day... I'll learn how to create an eslint rule!! fixed :muscle: